src/Client.php

```diff
@@ -644,7 +694,9 @@ public function updateEmailContent($emailId, $args = array(), $returnRaw = false
     {
         $args['id'] = $emailId;
 
-        return $this->getResult('updateEmailContent', $args, false, $returnRaw);
+        return new Response(
+            json_decode($this->post(sprintf('rest/asset/v1/email/%d/content.json', $emailId), $args)->getBody())
```

Comment:
I do like the old pattern of returning `getResult` - that way we don't duplicate logic to get the right format

Reply:
but that can totally be ignored if this is not something we are meant to maintain ourselves in the long run!

Reply:
I can see about reusing `getResult`, but the older logic inside was particular to Guzzle 3 and using descriptions. I can take a look

Reply:
The older pattern won't work as well since we need to use `sprintf` to replace arguments in the URL, something that was done automatically in guzzle 3 with the use of descriptions

Reply:
I see - I would do something like this 

```
$url = sprintf('rest/asset/v1/email/%d/content.json';
return new Response($url, $args);
```

Reply:
then 
```
private function _post($url, $args) {
  return json_decode($this->post($url, $args)->getBody());
}
```

just an example - might wanna change the name etc

Reply:
I like the idea of `_post`, `_get`, etc. I'll get that updated